Objects/Objects/BuiltElements/Revit/RevitToposolid.cs

```diff
@@ -0,0 +1,37 @@
+using System.Collections.Generic;
+using Objects.Geometry;
+using Speckle.Core.Kits;
+using Speckle.Core.Models;
+
+namespace Objects.BuiltElements.Revit
+{
+  public class RevitToposolid : Base, IDisplayValue<List<Mesh>>
+  {
+    public RevitToposolid() { }
+
+    [SchemaInfo("SpeckleToposolid", "Creates a Speckle Toposolid", "BIM", "Architecture")]
```

Comment:
Change this to: "RevitToposolid", "Creates a Revit Toposolid, "Revit", "Architecture"